utils/threshAlgos.py

```diff
@@ -1225,15 +1262,18 @@ def sbitRateAnalysis(chamber_config, rateTree, cutOffRate=0.0, debug=False, outf
     from gempython.utils.wrappers import runCommand
     from gempython.gemplotting.utils.anautilities import getDirByAnaType
     for entry in crateMap:
-        ohKey = (entry['shelf'],entry['slot'],entry['link'])
+        if hasDetNameBranch:
+            detName = entry['detName'][0]
+        else:
+            detName = chamber_config[(entry['shelf'],entry['slot'],entry['link'])]            
```

Comment:
Likewise for this block, used many times, usually the same syntax

Reply:
I have added a function to do this now.